sdk/tests/conformance/context/methods.html

```diff
@@ -183,6 +183,7 @@
 // Properties to be ignored because they were added in versions of the
 // spec that are backward-compatible with this version
 var ignoredMethods = [
+  "commit"
```

Comment:
Please add a comment saying from where this method comes: i.e., a link to the appropriate spec.